modules/node_modules/@ncigdc/uikit/Dropdown.js

```diff
@@ -35,7 +37,7 @@ const Dropdown = ({
         onMouseUp={mouseUpHandler}
         onClick={() => setTimeout(() => setActive(false))}
       >
-        {Children.map(children, child =>
+        {Children.map(_.filter(children, React.isValidElement), child =>
```

Comment:
You could use the native `filter` method here.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter